src/tools/rbd_wnbd/rbd_wnbd.cc

```diff
@@ -1210,7 +1232,12 @@ static int do_map(Config *cfg)
   int err = 0;
 
   if (g_conf()->daemonize && cfg->parent_pipe.empty()) {
-    return send_map_request(get_cli_args());
+    r = send_map_request(get_cli_args());
+    if (r) {
```

Comment:
```suggestion
    if (r < 0) {
```

Reply:
Out of caution, I'd check for any non-zero value here since we may end up forwarding a Windows status code.